src/Plugin.php

```diff
@@ -99,23 +116,14 @@ public function plugin_action_links( $actions ) {
 	 */
 	public function enabled_types() {
 
-		$stored_types     = get_option( 'file_upload_types', array() );
-		$enabled_types    = isset( $stored_types['enabled'] ) ? (array) $stored_types['enabled'] : array();
-		$custom_types_raw = isset( $stored_types['custom'] ) ? (array) $stored_types['custom'] : array();
+		$stored_types     = get_option( 'file_upload_types', [] );
+		$enabled_types    = isset( $stored_types['enabled'] ) ? (array) $stored_types['enabled'] : [];
+		$custom_types_raw = isset( $stored_types['custom'] ) ? (array) $stored_types['custom'] : [];
 		$available_types  = fut_get_available_file_types();
-		$return_types     = array();
-
-		foreach ( $available_types as $type ) {
-			if ( in_array( $type['ext'], $enabled_types, true ) ) {
-
-				$ext = trim( $type['ext'], '.' );
-				$ext = str_replace( ',', '|', $ext );
-
-				$return_types[ $ext ] = $type['mime'];
-			}
-		}
+		$return_types     = $this->add_available_types( [], $available_types, $enabled_types );
```

Comment:
I didn't get why we need 1st parameter as an empty array?
Could you explain your idea?

Reply:
sorry, removed. This function evaluates as I was writing it and at some point it made sense, no does not but I forgot to optimize.

Reply:
changed :)